FeedAPIChallenge/Feed API/RemoteFeedLoader.swift

```diff
@@ -19,6 +19,46 @@ public final class RemoteFeedLoader: FeedLoader {
 	}
 
 	public func load(completion: @escaping (FeedLoader.Result) -> Void) {
-		fatalError("Must be implemented")
+		client.get(from: url) { [weak self] result in
+			guard self != nil else { return }
+
+			switch result {
+			case let .success((data, response)):
+				completion(
+					Result { try map(data, response) }
+				)
+			case .failure:
+				completion(.failure(Error.connectivity))
+			}
+		}
+	}
+}
+
+private struct Root: Decodable {
+	let items: [DecodableFeedImgae]
+}
+
+private struct DecodableFeedImgae: Decodable {
```

Comment:
There's a typo, please fix it.

```suggestion
private struct DecodableFeedImage: Decodable {
```

Reply:
Thank you🙂 I fixed it.